securing the oracle.md

```diff
@@ -0,0 +1,13 @@
+---

```

Comment:
This is very similar to PR #137. We seem to be re-saying a lot of the same stuff. Not convinced we need this section but open to hearing your thoughts

Reply:
I suppose it was two different angles.

This is more focused on what happens as a result of liquidations (ie who gets what in which scenario), and the other was more about token holders responsibility when the DVM is called....but yeah, I do agree there is substantial overlap.